env/jun19/sim/spack.yaml

```diff
@@ -1,25 +1,27 @@
 spack:
   definitions:
-    - when: platform == 'linux'
-      root: [root@6.16.00+fortran+gdml+memstat+pythia6+pythia8+vc~vdt+python+tmva+xrootd+sqlite ^mesa~llvm]
-    - when: platform == 'darwin'
-      root: [root@6.16.00+fortran+gdml+memstat+pythia6+pythia8+vc~vdt+python+tmva+xrootd+sqlite+aqua]
+  - when: platform == 'linux'
+    root: [root@6.16.00+fortran+gdml+memstat+pythia6+pythia8+vc~vdt+python+tmva+xrootd+sqlite
+        ^mesa~llvm]
+  - when: platform == 'darwin'
+    root: [root@6.16.00+fortran+gdml+memstat+pythia6+pythia8+vc~vdt+python+tmva+xrootd+sqlite+aqua]
   specs:
-    - python@2.7.16
-    - py-numpy@1.16.5
-    - googletest@1.8.1
-    - boost@1.68.0
-    - fairlogger@1.4.0
-    - dds@2.4
-    - fairmq@1.4.3
-    - pythia6@428-alice1
-    - hepmc@2.06.09 length=CM momentum=GEV
-    - pythia8@8240
-    - geant4@10.05.p01~qt~vecgeom~opengl~x11~motif~data~clhep~threads
-    - $root
-    - geant3@2-7_fairsoft
-    - vgm@4-5
-    - geant4-vmc@4-0-p1
-    - fairroot@18.2.1+sim+examples
+  - python@2.7.18
+  - py-numpy@1.16.6
+  - py-setuptools@44.1.0
+  - googletest@1.8.1
+  - boost@1.68.0
+  - fairlogger@1.4.0
+  - dds@3.0
+  - fairmq@1.4.11
```

Comment:
I hope, that the switch from `dds@2.4` to `@3.0` does not introduce any incompatibilities? (a major version number is a heads up for me).

Reply:
fyi: https://github.com/FairRootGroup/DDS/blob/3.0/ReleaseNotes.md

There were changes, but they are all supported by FairMQ.

Reply:
From the Releases Notes you linked:
>  Modified: API breaking change! Names of all API headers are now streamlined.

So if an experiment/etc uses DDS directly, this could break?

Reply:
Yes, but no experiment uses DDS APIs directly except FairMQ.